tests/dataset_builders/pie/test_argmicro.py

```diff
@@ -0,0 +1,549 @@
+import json
+from collections import Counter
+from typing import List
+
+import pytest
+from datasets import disable_caching, load_dataset
+from pytorch_ie.core import Document
+from pytorch_ie.documents import (
+    TextBasedDocument,
+    TextDocumentWithLabeledSpansAndBinaryRelations,
+)
+from transformers import AutoTokenizer, PreTrainedTokenizer
+
+from dataset_builders.pie.argmicro.argmicro import (
+    ArgMicro,
+    ArgMicroDocument,
+    convert_to_text_document_with_labeled_spans_and_binary_relations,
+    document_to_example,
+    example_to_document,
+)
+from pie_datasets import DatasetDict
+from pie_datasets.document.conversion import tokenize_document
+from pie_datasets.document.types import TokenDocumentWithLabeledSpansAndBinaryRelations
+from tests import FIXTURES_ROOT
+from tests.dataset_builders.common import HF_DS_FIXTURE_DATA_PATH, PIE_BASE_PATH
+
+disable_caching()
+
+DATASET_NAME = "argmicro"
+SPLIT_SIZES = {"train": 112}
+DATA_PATH = FIXTURES_ROOT / "dataset_builders" / "arg-microtexts-master.zip"
+HF_DATASET_PATH = ArgMicro.BASE_DATASET_PATH
+PIE_DATASET_PATH = PIE_BASE_PATH / DATASET_NAME
+
+
+@pytest.fixture(scope="module", params=["en", "de"])
+def language(request):
+    return request.param
+
+
+@pytest.fixture(scope="module")
+def hf_dataset(language):
+    return load_dataset(str(HF_DATASET_PATH), name=language, data_dir=DATA_PATH)
+
+
+@pytest.fixture(scope="module")
+def generate_document_kwargs(hf_dataset, language):
+    ds = hf_dataset["train"]
+    return ArgMicro(config_name=language)._generate_document_kwargs(ds)
+
+
+def test_hf_dataset(hf_dataset, language, generate_document_kwargs):
+    assert hf_dataset is not None
+    assert {name: len(ds) for name, ds in hf_dataset.items()} == SPLIT_SIZES
+    ds = hf_dataset["train"]
+    # check

```

Comment:
check what?

Reply:
Not sure when it got there but deleted in cced93b